packages/twitch-pubsub-client/src/Messages/PubSubUndocumentedMessage.ts

```diff
@@ -0,0 +1,33 @@
+import { Enumerable } from '@d-fischer/shared-utils';
+import { rtfm } from 'twitch-common';
+
+/** @private */
+export interface PubSubUndocumentedMessageContent {
+	data: unknown;
+}
+
+/** @private */
+export interface PubSubUndocumentedMessageData {
+	data: PubSubUndocumentedMessageContent;
+	type: string;
```

Comment:
Is there really a `type` property in *every* message? Also, can we assume *anything* about topics we don't know? I'd prefer just using the `unknown` type for the whole structure and returning that whole thing in the `data` getter.